rules-reviewed/eap7/eap72/maven-javax-to-jakarta.rhamt.xml

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="maven-javax-to-jakarta"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Maven built applications that depend on Java EE artifacts, which were
+            replaced by Jakarta EE artifacts, starting with JBoss EAP 7.3.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[7,8)" />  
+        <targetTechnology id="jakarta-ee" versionRange="[8]" />      
+    </metadata>
+    <rules>
+        <!-- https://issues.jboss.org/browse/WINDUPRULE-396 -->
+        <!-- com.sun.mail:javax.mail -> com.sun.mail:jakarta.mail -->
+        <rule id="maven-javax-to-jakarta-00001">
+            <when>
+                <project>
+                    <artifact groupId="com.sun.mail" artifactId="javax.mail" />
+                </project>
+            </when>
+            <perform>
+                <hint title="Move to Jakarta EE Maven Artifacts - com.sun.mail:javax.mail" effort="1" category-id="potential">
+                    <message>If you migrate your application to JBoss EAP 7.3.x, and want to ensure its Maven building, running or testing works as expected, use instead the Jakarta EE dependency with groupId `com.sun.mail`, and artifactId `jakarta.mail`.</message>
```

Comment:
To be consistent with the changes requested in other rules' messages:
```suggestion
                    <message>If you migrate your application to JBoss EAP 7.3 (or later), and want to ensure its Maven building, running or testing works as expected, use instead the Jakarta EE dependency with groupId `com.sun.mail`, and artifactId `jakarta.mail`.</message>
```

Reply:
done